### PR TITLE
BRITECH-194 Added front end to cloud formation template in new cluster

### PR DESCRIPTION
**Pull request by thullSL**
Front end is on a separate cluster with it's on EC2 instance

### CODE REVIEW

File: dataCenter/singleEcs.json [hunk 1]
```diff
@@ -2,12 +2,19 @@
...
     "Description" : "A stack for the non-persistent, backend infrastructure used by BristechSRM",
     "Parameters" : {
-        "PublicElasticIP" : {
+        "apiElasticIP" : {
             "Type" : "String",
             "MinLength" : "4",
             "MaxLength" : "15",
             "Default" : "null",
-            "Description" : "The Elastic IP to associate with this stack. Enter null to skip having public IP assignment"
+            "Description" : "The Elastic IP to associate with this stacks API resocources. Enter null to skip having public IP assignment"
```
Comment (lclarksl) [inline dataCenter/singleEcs.json]: stack's

resources

File: dataCenter/singleEcs.json [hunk 2]
```diff
@@ -2,12 +2,19 @@
...
+        },
+        "frontElasticIP" : {
+            "Type" : "String",
+            "MinLength" : "4",
+            "MaxLength" : "15",
+            "Default" : "null",
+            "Description" : "The Elastic IP to associate with this stacks front end. Enter null to skip having public IP assignment"
```
Comment (lclarksl) [inline dataCenter/singleEcs.json]: stack's

File: dataCenter/singleEcs.json [hunk 3]
```diff
@@ -85,9 +98,88 @@
...
+                    {"Key": "Purpose","Value": "docker"}
+                ],
+                "NetworkInterfaces": [
+                    {
+                        "DeleteOnTermination": "true",
+                        "DeviceIndex": 0,
+                        "SubnetId": "subnet-b64844c1",
+                        "GroupSet": [
+                            "sg-b6e9d4d2"
+
```
Comment (lclarksl) [inline dataCenter/singleEcs.json]: Empty line looks weird here

**Comment by lclarksl** [comment]
:shipit: after typos

**Comment by dwybourn** [comment]
:shipit:

**Pull request merged**